projects/plugins/boost/tests/e2e/specs/base/jetpack-compatibility.test.js

```diff
@@ -1,94 +1,23 @@
 import { test, expect } from 'jetpack-e2e-commons/fixtures/base-test.js';
 import { JetpackBoostPage } from '../../lib/pages/index.js';
-import { prerequisitesBuilder, isModuleActive } from 'jetpack-e2e-commons/env/prerequisites.js';
-import { activateModules, deactivateModules } from 'jetpack-e2e-commons/env/index.js';
-import {
-	boostPrerequisitesBuilder,
-	activateModules as activateBoostModules,
-	deactivateModules as deactivateBoostModules,
-} from '../../lib/env/prerequisites.js';
+import { prerequisitesBuilder } from 'jetpack-e2e-commons/env/prerequisites.js';
+import { activateModules } from 'jetpack-e2e-commons/env/index.js';
 
 test.describe( 'Jetpack compatibility', () => {
 	test.beforeAll( async () => {
 		await prerequisitesBuilder().withActivePlugins( [ 'jetpack' ] ).build();
 	} );
 
-	test( 'The Jetpack lazy-image module activation reflects in Boost dashboard', async ( {
+	test( 'The Jetpack lazy-image module can not be activated in Boost dashboard', async ( {
 		page,
 	} ) => {
 		await prerequisitesBuilder( page ).withInactiveModules( [ 'lazy-images' ] ).build();
 		await activateModules( [ 'lazy-images' ] );
 
```

Comment:
I think lazy loading toggle tests should be removed for now until the module is fully removed.

Reply:
Good idea. I wasn't sure if I should leave it or remove it. I guess it will only hurt us having them here.

I've removed them in fd2e8fb and updated other tests depending on lazy loading in 5f3e766.

I also noticed out tests are outdated in general, so I opened a ticker for that https://github.com/Automattic/jetpack/issues/33877